### PR TITLE
Update register-sso-add-in-aad-v2.md

### PR DESCRIPTION
**Pull request by mattgeim**

### CODE REVIEW

File: docs/develop/register-sso-add-in-aad-v2.md [hunk 1]
```diff
@@ -18,4 +18,7 @@ The following table itemizes the information that you need to carry out this pro
 |The fully qualified domain name (except for protocol) of the add-in. *You must use a domain that you own.* For this reason, you cannot use certain well-known domains such as `azurewebsites.net` or `cloudapp.net`. The domain must be the same, including any subdomains, as is used in the URLs in the **\<Resources\>** section of the add-in's manifest.|`localhost:6789`, `addins.contoso.com`|`<fully-qualified-domain-name>`|
 |The permissions to the Microsoft identity platform and Microsoft Graph that your add-in needs. (`profile` is always required.)|`profile`, `Files.Read.All`|N/A|
 
-[!INCLUDE [register-sso-add-in-aad-v2-include](../includes/register-sso-add-in-aad-v2-include.md)]
\ No newline at end of file
+> [!IMPORTANT]
```
Comment (davidchesnut) [inline docs/develop/register-sso-add-in-aad-v2.md:21]:
```suggestion
> [!CAUTION]
```

Use caution for potential negative consequences.

File: docs/develop/register-sso-add-in-aad-v2.md [hunk 2]
```diff
@@ -18,4 +18,7 @@ The following table itemizes the information that you need to carry out this pro
...
+> **Sensitive information**: The application ID URI (`<fully-qualified-domain-name>`) is logged as part of the authentication process when an add-in using SSO is activated in Office running inside of Microsoft Teams and mustn't contain sensitive information.
```
Comment (davidchesnut) [inline docs/develop/register-sso-add-in-aad-v2.md:22]:
```suggestion
> **Sensitive information**: The application ID URI (`<fully-qualified-domain-name>`) is logged as part of the authentication process when an add-in using SSO is activated in Office running inside of Microsoft Teams. The URI mustn't contain sensitive information.
```

**Review by davidchesnut — CHANGES_REQUESTED** [review]
Looks good, just a couple suggestions, thanks!

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 3]
```diff
@@ -117,7 +117,7 @@ Sometimes called an _application password_, a client secret is a string value yo
 
     The **Add a client application** pane appears.
 
-1. In the **Client ID** enter `ea5a67f6-b6f3-4338-b240-c655ddc3cc8e`. This value pre-authorizes all Microsoft Office application endpoints.
+1. In the **Client ID** enter `ea5a67f6-b6f3-4338-b240-c655ddc3cc8e`. This value pre-authorizes all Microsoft Office application endpoints. If you also would like to pre-authorize Office when used inside of Microsoft Teams, add `1fec8e78-bce4-4aaf-ab1b-5451cc387264` ( Teams desktop and Teams Mobile) and `5e3ce6c0-2b1f-4285-8d4b-75ee78787346` (Teams on the web).
```
Comment (davidchesnut) [inline docs/includes/register-sso-add-in-aad-v2-include.md:120]:
```suggestion
1. In the **Client ID** enter `ea5a67f6-b6f3-4338-b240-c655ddc3cc8e`. This value pre-authorizes all Microsoft Office application endpoints. If you also want to pre-authorize Office when used inside of Microsoft Teams, add `1fec8e78-bce4-4aaf-ab1b-5451cc387264` (Microsoft Teams desktop and Teams mobile) and `5e3ce6c0-2b1f-4285-8d4b-75ee78787346` (Teams on the web).
```

**Review by davidchesnut — CHANGES_REQUESTED** [review]
A few suggested changes. Thanks!

**Review by mattgeim — COMMENTED** [review]
.

**Pull request merged**